### PR TITLE
ENH Dynamically use correct branch of supported-modules

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/22

### CODE REVIEW

File: src/Translator.php [hunk 1]
```diff
@@ -119,10 +119,20 @@ private function scanDir(string $dir): array
         return array_diff(scanDir($dir), ['.', '..']);
     }
 
+    private function getCmsMajor(): string
```
Comment (michalkleiner) [inline src/Translator.php]:
```suggestion
    private function getCmsFrameworkMajor(): string
```
or
```suggestion
    private function getFrameworkMajor(): string
```
or 
query `composer show silverstripe/cms` ?

Comment (emteknetnz) [reply src/Translator.php]: Updated

File: src/Translator.php [hunk 2]
```diff
@@ -119,10 +119,20 @@ private function scanDir(string $dir): array
         return array_diff(scanDir($dir), ['.', '..']);
     }
 
+    private function getFrameworkMajor(): string
+    {
+        $output = $this->exec('composer show silverstripe/framework | grep versions');
+        if (preg_match('#^versions : \* ([4-9])#', $output, $matches)) {
+            return $matches[1];
+        }
+        return '5';
```
Comment (ChloeSartorelli) [inline src/Translator.php]: We should throw a `LogicException` if we don't get a match - it indicates that we've done something wrong either with the setup of our environment or in the code for the translator.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: src/Translator.php (hunk 2 above)
Comment (emteknetnz) [reply src/Translator.php]: Updated

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**